airbyte/cli.py

```
@@ -0,0 +1,341 @@
+)
+@click.option(
+    "--job-file",
+    type=str,
+    help="A yaml file containing the job definition.",
+)
+@click.option(
+    "--job-dpath",
+    type=str,
+    help="The dpath expression pointing to a job definition within the job file.",
+)
+def benchmark(
```

Ruff

Missing argument descriptions in the docstring for `benchmark`: `config`, `job_dpath`, `job_file`

(D417)